ignition/resources/worker-kubelet.service

```diff
@@ -0,0 +1,52 @@
+[Unit]
+After=get-ssl.service
+Requires=get-ssl.service
+[Service]
+Environment=KUBELET_IMAGE_URL=${kubelet_image_url}
+Environment=KUBELET_IMAGE_TAG=${kubelet_image_tag}
+Environment="RKT_RUN_ARGS=\
+  --uuid-file-save=/var/run/kubelet-pod.uuid \
+  --volume var-log,kind=host,source=/var/log \
+  --mount volume=var-log,target=/var/log \
+  --volume cni-bin,kind=host,source=/opt/cni/bin \
+  --mount volume=cni-bin,target=/opt/cni/bin \
+  --volume var-lib-cni,kind=host,source=/var/lib/cni \
+  --mount volume=var-lib-cni,target=/var/lib/cni \
+  --volume etc-cni-netd,kind=host,source=/etc/cni/net.d \
+  --mount volume=etc-cni-netd,target=/etc/cni/net.d \
+  --volume dns,kind=host,source=/etc/resolv.conf \
+  --mount volume=dns,target=/etc/resolv.conf \
+  --volume modprobe,kind=host,source=/usr/sbin/modprobe \
+  --mount volume=modprobe,target=/usr/sbin/modprobe \
+  --volume lib-modules,kind=host,source=/lib/modules \
+  --mount volume=lib-modules,target=/lib/modules"
+ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
+ExecStartPre=/usr/bin/mkdir -p /var/log/containers
+ExecStartPre=/usr/bin/mkdir -p /opt/cni/bin
+ExecStartPre=/usr/bin/mkdir -p /var/lib/cni
+ExecStartPre=/usr/bin/mkdir -p /etc/cni/net.d
+ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/run/kubelet-pod.uuid
+ExecStart=/usr/lib/coreos/kubelet-wrapper \
+  --kubeconfig=/var/lib/kubelet/kubeconfig \
+  --require-kubeconfig=true \
+  --node-labels=role=${role} \
+  --container-runtime=docker \
+  --network-plugin-dir=/etc/cni/net.d \
+  --network-plugin=cni \
+  --cni-bin-dir=/opt/cni/bin \
+  --cni-conf-dir=/etc/cni/net.d \
+  --register-node=true \
+  --allow-privileged=true \
+  --pod-manifest-path=/etc/kubernetes/manifests \
+  --cluster-dns=${cluster_dns} \
+  --cluster_domain=cluster.local \
+  ${cloud_provider == "" ? "" : "--cloud-provider=${cloud_provider}"} \
+  --eviction-soft=memory.available<2Gi,nodefs.available<4Gi \
+  --eviction-soft-grace-period=memory.available=1m,nodefs.available=1m \
+  --eviction-max-pod-grace-period=30 \
+  --eviction-hard=memory.available<1Gi,nodefs.available<2Gi
```

Comment:
Potentially. For now, I've tried to resist changing things like this or exposing more configuration than what's required to test this setup. These values are the same across all our current environments so I thought we could keep it as-is.

There are, however, a few items that could be exposed as configuration values to the caller of the module. It really depends on how we end up using this module.